appveyor.yml

```diff
@@ -7,6 +7,10 @@ matrix:
   fast_finish: true
 
 environment:
+  BOLT_WINRM_USER: roddypiper
+  BOLT_WINRM_PASSWORD: r0Wdy999
```

Comment:
What happened to the commit that generated a secure password?

Reply:
I removed for the sake of simplicity / being more declarative, but can easily add back random password generation